pkg/controller/ads/ads_processor.go

```diff
@@ -184,12 +184,6 @@ func (p *processor) handleCdsResponse(resp *service_discovery_v3.DiscoveryRespon
 	// Note eds typed cluster, we do not flush to bpf map here, we need to wait for eds update.
 	p.Cache.ClusterCache.Flush()
 
-	if p.lastNonce.edsNonce == "" {
```

Comment:
why delete this?

Reply:
This case can be solved after the reset edsClusterNames command is executed.